### PR TITLE
Fix inverted explanation of tight lists

### PR DESCRIPTION
**Pull request by robjwells**
At present, the syntax reference documents tight lists as _containing_ blank lines, which is the opposite of the example (and what one would expect of "tight" lists).

This change would reword the explanation to say that tight lists do not have blank lines between items or blocks inside an item.

### CODE REVIEW

**Comment by robjwells** [comment]
Second push just fixes a typo (`lnes` -> `lines`) in the commit message body.

**Comment by jgm** [comment]
Many thanks!
